src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -0,0 +1,173 @@
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.CommandLineUtils;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Packaging.Signing;
+using static NuGet.Commands.TrustedSignersArgs;
+
+namespace NuGet.CommandLine.XPlat.Commands.Signing
+{
+    internal static class TrustedSignersCommand
+    {
+        public enum TrustCommand
+        {
+            List,
+            Author,
+            Repository,
+            Source,
+            Certificate,
+            Remove,
+            Sync
+        }
+
+        internal static void Register(CommandLineApplication app,
+                      Func<ILogger> getLogger,
+                      Action<LogLevel> setLogLevel)
+        {
+            app.Command("trust", trustedSignersCmd =>
+            {
+                CommandArgument command = trustedSignersCmd.Argument(
+                    "command",
+                    Strings.TrustCommandActionDescription,
+                    multipleValues: true);
+
+                CommandOption algorithm = trustedSignersCmd.Option(
+                    "--algorithm",
+                    Strings.TrustCommandAlgorithm,
+                    CommandOptionType.SingleValue);
+
+                CommandOption allowuntrustedrootOption = trustedSignersCmd.Option(
+                    "--allow-untrusted-root",
+                    Strings.TrustCommandAllowUntrustedRoot,
+                    CommandOptionType.NoValue);
+
+                CommandOption owners = trustedSignersCmd.Option(
+                    "--owners",
+                    Strings.TrustCommandOwners,
+                    CommandOptionType.MultipleValue);
+
+                CommandOption verbosity = trustedSignersCmd.Option(
+                    "-v|--verbosity",
+                    Strings.Verbosity_Description,
+                    CommandOptionType.SingleValue);
+
+                CommandOption configfile = trustedSignersCmd.Option(
+                    "--configfile",
+                    Strings.Option_ConfigFile,
+                    CommandOptionType.SingleValue);
+
+                trustedSignersCmd.HelpOption(XPlatUtility.HelpOption);
+                trustedSignersCmd.Description = Strings.TrustCommandDescription;
+
+                trustedSignersCmd.OnExecute(async () =>
+                {
+                    TrustCommand action;
+
+                    if (!command.Values.Any() || string.IsNullOrEmpty(command.Values[0]))
+                    {
+                        action = TrustCommand.List;
+                    }
+                    else if (!Enum.TryParse(command.Values[0], ignoreCase: true, result: out action))
+                    {
+                        throw new CommandLineArgumentCombinationException(string.Format(CultureInfo.CurrentCulture, Strings.Error_UnknownAction, command.Values[0]));
+                    }
+
+                    string name = null;
+                    if (command.Values.Count() > 1)
```

Comment:
`CommandArgument.Values` is a `List`.
```suggestion
                    if (command.Values.Count > 1)
```

Reply:
Addressed.